lib/vehicle_factory.rb

```diff
@@ -0,0 +1,13 @@
+class VehicleFactory
+  def create_vehicles(registrations)
+    auto = registrations.map do |registration|
```

Comment:
Nice work extracting and parsing this data! You actually don't even need to set it to the variable `auto`. It can just live as it is with the .map and that will return the collection of Vehicles